scripts/terminalserver/terminalserver.jl

```diff
@@ -35,6 +35,56 @@ function sendDisplayMsg(kind, data)
 end
 
 
+function is_module_loaded(mod)
+    if mod == "Main"
+        return true
+    end
+
+    ms = split(mod, '.')
+
+    out = Main
+
+    loaded_module = findfirst(==(first(ms)), string.(Base.loaded_modules_array()))
+
+    if loaded_module !== nothing
+        out = Base.loaded_modules_array()[loaded_module]
+        popfirst!(ms)
+    end
+
+    for m in Symbol.(ms)
+        if isdefined(out, m)
+            resolved = getfield(out, m)
+
+            if resolved isa Module
+                out = resolved
+            else
+                return out
```

Comment:
maybe you want 
```suggestion
                return out !== Main
```
?

Reply:
Yeah, good catch.